spec/lib/tasks/bulk_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe "bulk" do
+  describe "email" do
+    before do
+      Rake::Task["bulk:email"].reenable
+    end
+
+    it "sends emails to a subscription list" do
+      subscriber_list = create(:subscriber_list)
+
+      ClimateControl.modify(SUBJECT: "subject", BODY: "body") do
```

Comment:
(You can also do this in an `around` block.)

Reply:
[👍 ](https://github.com/alphagov/email-alert-api/commit/500fd67f01c9d5005dd49870459ba8c195ef0b05)